brainstormforce/astra
# AST-1854 - Blog Achieve layout 3 broken between 769px to 921px

### Description
There are some issues with the Blog Archive layout 3 between 769px and 921px width.

Issue with Layout 3The layout is breaking when the container is Boxed / Content Boxed.

This is how it displays between 769px - 921px: https://d.pr/i/V9DTiu.


### Screenshots
https://share.bsf.io/12ugWek2

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
How to reproduce:
- Activate the blog pro module and select or Layout 3
- Check the frontend between 769px - 921px
- For Layout 3, you will notice the weird UI -  https://share.bsf.io/12ugWek2

### Checklist:
